src/components/organisms/Footer.tsx

```diff
@@ -4,7 +4,7 @@ const Footer = () => {
   return (
     <footer className="center-col rounded-t-xl w-full gap-4 bottom py-10  bg-green">
       <SocialMedia />
-      <p className="md:text-small text-xs text-center px-8 max-w-[30rem]">
+      <p className="text-white md:text-small text-xs text-center px-8 max-w-[30rem]">
```

Comment:
This does the work, but the downside of this approach is that we will need to add text-white to all of our future components. I believe a better way to do this is by changing body color style on main global.css file